pulser/sequence.py

```diff
@@ -1183,18 +1177,19 @@ def config_slm_mask(self, qubits: Set[QubitId]) -> None:
 
         # Find tentative initial and final time of SLM mask if possible
         for channel in self._channels:
-            # Cycle on slots in schedule until the first pulse is found
-            for slot in self._schedule[channel]:
-                if not isinstance(slot.type, Pulse):
-                    continue
-                ti = slot.ti
-                tf = slot.tf
-                if self._slm_mask_time:
-                    if ti < self._slm_mask_time[0]:
+            if self._channels[channel].addressing == "Global":
+                # Cycle on slots in schedule until the first pulse is found
+                for slot in self._schedule[channel]:
```

Comment:
This is just minutia, but it will make the code less pushed to the right, which is usually a good thing:
```suggestion
            if not self._channels[channel].addressing == "Global":
                continue
            # Cycle on slots in schedule until the first pulse is found
            for slot in self._schedule[channel]:
```